tools/tcp_utilities/plus_tcp_demo_cli.c

```diff
@@ -945,7 +945,7 @@ static struct freertos_addrinfo * pxDNSLookup( char * pcHost,
 
                 #if ( ipconfigUSE_IPv6 != 0 )
                     {
-                        pxResult->ai_addr = ipPOINTER_CAST( struct freertos_sockaddr *, &( pxResult->xPrivateStorage.sockaddr6 ) );
+                        pxResult->ai_addr = ipPOINTER_CAST( struct freertos_sockaddr *, & ( pxResult->xPrivateStorage.sockaddr6 ) );
```

Comment:
Uh? I have seen this space before:
~~~c
- &( pxResult->xPrivateStorage.sockaddr6 )
+ & ( pxResult->xPrivateStorage.sockaddr6 )
~~~